File: goproject/src/exercise14/main/time_demo.py
import time


def test():
	s = ''
	for i in range(100000):
		s += 'hello' + str(i)


if __name__ == '__main__':
	start = time.time()
	test()
	end = time.time()
	print(f"(python)耗时={end - start}s")

File: goproject/src/exercise14/main/time_demo.go
package main
import (
	"fmt"
	"strconv"
	"time"
)


func test() {
	s := ""
	for i := 0; i < 100000; i++ {
		s += "hello" + strconv.Itoa(i)
	}
}


func main() {
	start := time.Now().Unix()
	test()
	end := time.Now().Unix()
	fmt.Printf("(go)耗时=%vs\n", end - start)
}